### ISSUE TITLE
I don't know where to ask but I can't open my Trinity Wallet with this error.

### ISSUE THREAD

**Issue by GCiota** [iotaledger/trinity-wallet#2779]
## Bug description

Can't open the wallet

## Version

Which version of Trinity are you running? (1.4.5)

- Version: (1.4.5)

## Hardware specification

What hardware are you using?

- Mobile or Desktop: Mac Book Pro
- Device:
- OS and version: Catalina 10.15.4

- RAM (Desktop only): 16GB
- Number of cores (Desktop only): 6 cores

## Node Settings (delete if not relevant)

What are your node settings? Are you using automatic node management?

- Node settings: 

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

1. 
2. 
3. 

## Expected behaviour

Describe what you expect to happen.
I'd like to open my wallet. 
## Actual behaviour

Describe what actually happens.

## Errors

When I try to open the wallet, it doesn't open with the following error message.

Error Launching Wallet
There was a fatal error launching the wallet.
Unable to open realm at path ' /Users/ (my computer name) / Library / Application / Support / Trinity / Trinity.realm' Realm file decryption failed Path:

**Comment by cvarley100** [comment]
Hi @GCiota it sounds like your database has been corrupted somehow. Easiest solution is to reset the wallet and re-enter your seed(s) from backup.

**Comment by GCiota** [comment]
@cvarley100 Thank you so much for your time. May I check a few things before I move any forward. 

1) How do you reset the wallet? Do you simply suggest to delete the wallet and reinstall?

2) I haven't actually done the backup process since I installed my Trinity almost a year and half ago. Would my 16 back up letter be enough to recover my fund? Or is there anything else that I'm supposed to have other than that 16 back up letter? I just wanna make sure if I have things ready before deleting anything.

3) Lastly, I think this problem begins after I changed my laptop password. Does it mean I can fix it by just changing my laptop pwd back to what it was?

Thank you so much for your time.
G

**Comment by GCiota** [comment]
Re 2 - I meant to say "would my 24 backup phrase of Ledger Nano S be enough to recover?

**Comment by cvarley100** [comment]
@GCiota If your funds are on a Ledger, all you need is the index number of your account (usually 0 but you might have chosen a different one). You can delete the Trinity folder in Application Support - `~/Library/Application Support/Trinity`. You may need to unhide hidden folders.

**Comment by GCiota** [comment]
@cvarley100 Hi, thank you for your time again. So all I need is just re install the wallet after deleting it?

**Comment by GCiota** [comment]
@cvarley100 I know I'm being too cautious but I'm really scared to lose my funds so. Glad to hear that my fund should be safe if I used Ledger, which I did. Thanks

**Comment by cvarley100** [comment]
@GCiota Your seed is stored in the Ledger device, not the Trinity application. So long as you are confident you know your Ledger index number (usually 0), you're good to go.

**Comment by GCiota** [comment]
@GCiota copy that. Thank you for the confirmation.

**Comment by cvarley100** [comment]
Did you resolve this @GCiota?

**Comment by GCiota** [comment]
@cvarley100 Thanks for checking. Yes, I deleted Trinity folder in Library and reset up the wallet with the Ledger. And when i put my index number, it worked and my funds are still there.

Thank you so so so much!

**Comment by traminal** [comment]
How to reset the wallet in the Ledger? I am having the same problem. I deleted the Trinity wallet as suggested but it still didnt work. I think I need to reset the Ledger as well. Can you show me what you did to fix this problem. Thanks!

**Comment by GCiota** [comment]
@traminal Hi there, I simply deleted and reinstalled the Trinity Wallet. Supposedly they should ask your "Ledger Index Number" when re-installing. I'm not sure what stage I was asked, but think it was somewhere between choosing "I already have a seed" and "creating a new seed"

So when they asked my "Ledger Index Number" I typed my number (Usually 0, but I had mine differently.) and was able to see my fund on the app. 

Check this tutorial too. It's helpful. 
https://www.youtube.com/watch?v=aPGkZQMPxKQ
